hhs_oauth_server/settings/aws-dev.py

```diff
@@ -44,9 +44,22 @@
 DEV_SPECIFIC_APPS = [
     # Installation/Site Specific apps based on  -----------------
     'storages',
+    # A test client - moved to aws-test / dev /impl settings
+    'apps.testclient',
+
 ]
 INSTALLED_APPS += DEV_SPECIFIC_APPS
 
+# Optional Apps will be loaded by environment.
+# We need to be able to check in the urls.py and in html so that we only call
+# active reverse matches.
+# Therefore add name of optional apps here.
+# then add to this list in the settings file for the environment.
+# every optional app should have their name added to this list
+# Then this variable will be added to SETTINGS_EXPORT below
+# so we can test in html templates
```

Comment:
Can we cut out this comment, or at least shorten? Our code has gotten very difficult to read due to the bulk of comments, this one is already inaccurate. I think this code is fairly self explanatory.